app/controllers/pages_controller.rb

```diff
@@ -0,0 +1,3 @@
+class PagesController < ApplicationController
+  def index; end
```

Comment:
Fun Fact: this is technically not needed. but being explicit and not relying on too much rails magic is good.

I just wanted to point out that rails will figure out that it needs to render the index.html.erb file if there is no index action defined